### PR TITLE
Closing #6: Max Resilience

### PR DESCRIPTION
**Pull request by jimmyygit**
image

I also made a Title component so you guys can reuse it

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
+
+            <h2 className="text-success font-weight-bold"> External Resilience </h2>
+            <p> The ability to define your role within your community. Being active in your community can help create your own identity. </p>
+
+            <Button type="purpleWhite" link="/">
+              <p>LEARN MORE</p>
+            </Button>
+          </div>
+        </div>
+      </section>
```
Comment (functionstackx) [inline pages/index.js]: can you do component?

**Review by functionstackx — CHANGES_REQUESTED** [review]
do component to clean up our codebase.

pages/ files should only be used to import components and hydrate components

File: pages/index.js [hunk 2]
```diff
@@ -1,13 +1,43 @@
...
 import Jumbotron from '../components/Jumbotron'
+import Title from '../components/Title'
+import Button from '../components/Button'
 
 export default function Home() {
   return (
     <>
       <Landing />
       <Jumbotron />
       <h1> Hello World </h1>
```
Comment (functionstackx) [inline pages/index.js]: can this hello world placeholder be removed

**Comment by jimmyygit** [comment]
I am happy to announce that this issue has been resolved.

**Pull request merged**